pkgs/development/tools/build-managers/bloop/default.nix

```diff
@@ -16,12 +16,12 @@ let
     '';
     outputHashMode = "recursive";
     outputHashAlgo = "sha256";
-    outputHash     = "0w2gk9ladkzm3mx3qsmjqjakj94g7s4s3xfzl6yny1h4dg1456n6";
+    outputHash     = "0ki4xh8z31nv42jik998r7gvsxcm0vysx83w0nyvbhi18cah1qh6";
   };
 in
 stdenv.mkDerivation rec {
   name = "${baseName}-${version}";
-  nailgunCommit = "6992a3bf";
+  nailgunCommit = "dc1dd806";
```

Comment:
I’m curious — where does this hash come from?

Reply:
From the official installation script https://github.com/scalacenter/bloop/releases/download/v1.1.2/install.py

Reply:
Ah, cool. Would you mind adding a comment saying that, so somebody who doesn’t know the package would be able to update it more easily?

Reply:
Ah, cool. Would you mind adding a comment saying that, so somebody who doesn’t know the package would be able to update it more easily?

Reply:
Done

Reply:
Thanks!